pkgs/tools/networking/openssh/default.nix

```diff
@@ -41,15 +43,7 @@ stdenv.mkDerivation rec {
       ./dont_create_privsep_path.patch
 
       ./ssh-keysign.patch
```

Comment:
what happened to `/detect_apple_ld.patch` from https://github.com/NixOS/nixpkgs/pull/81368 here?

Reply:
This patch was part of libfido2 package (added in other PR), not OpenSSH (this PR).

Reply:
Okay, thanks!